lib/msf/ui/console/command_dispatcher/nop.rb

```diff
@@ -59,7 +59,7 @@ def cmd_generate(*args)
           length = val.to_i
         when '-b'
           badchars = Rex::Text.dehex(val)
-        when "-c"
+        when "-s"
```

Comment:
Rather than substitute, lets add the -s option and keep the -c for a bit for compatibility.
```suggestion
when "-s", "-c"  # 'c' is deprecated; remove later
  saveregs = val.split(/,\s?/)
```